source/simulator/solver.cc

```diff
@@ -983,6 +1142,9 @@ namespace aspect
                           distributed_stokes_rhs,
                           preconditioner_cheap);
 
+
+
+
```

Comment:
same here adding this many empty lines does not follow our usual code style.

As a more general comment: If you find code that would want to improve with changes that are unrelated to your PR topic, please do not add the changes to the current PR, but instead open an new one.